### ISSUE TITLE
[Bug]:  mo-service startup panic after reboot

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#5676]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):7108574f2ca9b3c801faa17bcb11372e1d005598
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

panic: internal error: driver info: retry time out

goroutine 262 [running]:
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver.(*LogServiceDriver).readFromLogService(0xc00c70fcc0, 0xfb, 0xc00276f200?)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver/read.go:159 +0x270
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver.(*LogServiceDriver).readFromLogServiceInReplay(0xc00276f280?, 0x1b97245?, 0x2444d73?, 0xc00276f248)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver/read.go:129 +0x2f
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver.(*replayer).readRecords(0xc00defba40)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver/replay.go:82 +0x48
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver.(*replayer).replay(0xc00defba40)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver/replay.go:65 +0x3f
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver.(*LogServiceDriver).Replay(0xc00c70fcc0, 0x203003?)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver/driver.go:109 +0x33
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/logstore/store.(*StoreImpl).Replay(0xc0110acdc0, 0xc00defd278)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/logstore/store/replay.go:22 +0xaa
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/wal.(*walDriver).Replay(0xc000571d00, 0x0?)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/wal/driver.go:101 +0x37
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/db.(*Replayer).Replay(0xc00defb860)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/db/replay.go:89 +0x6d
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/db.(*DB).Replay(0xc008308000, 0xc00d5c8840)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/db/db.go:112 +0x1ef
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/db.Open({0xc00040a430, 0xb}, 0x0?)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/vm/engine/tae/db/open.go:91 +0x866
github.com/matrixorigin/matrixone/pkg/cnservice.initTAE({0x27f4110?, 0xc0004bb000}, 0xc0005d6a50, 0xc0001c0000)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/cnservice/tae.go:70 +0x325
github.com/matrixorigin/matrixone/pkg/cnservice.(*service).initEngine(0x245a8d6?, {0xc0004babc0?, 0x0?}, {0x27f4110?, 0xc0004bb000?}, 0x50?)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/cnservice/server.go:219 +0x66
github.com/matrixorigin/matrixone/pkg/cnservice.(*service).initMOServer(0xc0002b83c0, {0x27f4110, 0xc0004babc0}, 0xc0005d6a50)
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/cnservice/server.go:200 +0x117
github.com/matrixorigin/matrixone/pkg/cnservice.NewService(0xc0001c0000, {0x27f4110, 0xc0004babc0}, {0x2801720?, 0xc0003ca6f0}, {0x282c0c0?, 0xc005f24960}, {0xc00276f9a8, 0x1, 0x1})
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/cnservice/server.go:91 +0x4f8
main.startCNService.func1({0x27f4110, 0xc0004babc0})
	/mnt/vdb/dengnan/matrixone_cn/matrixone/cmd/mo-service/main.go:139 +0x26e
github.com/matrixorigin/matrixone/pkg/common/stopper.(*Stopper).doRunCancelableTask.func1()
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/common/stopper/stopper.go:257 +0x79
created by github.com/matrixorigin/matrixone/pkg/common/stopper.(*Stopper).doRunCancelableTask
	/mnt/vdb/dengnan/matrixone_cn/matrixone/pkg/common/stopper/stopper.go:252 +0x12c

we need to delete all data folders to successfully reboot mo-service.

**Comment by XuPeng-SH** [comment]
same issue as #5669

**Comment by sukki37** [comment]
tracked by #5669

**Comment by volgariver6** [comment]
resolved by #6012

**Comment by aressu1985** [comment]
fixed:1feeb04254c6fdab0ae0061ecc32349136d6ee10

no this panic any more, but still can not restart ,traced by #6284
